pkg/operator/controller/ingress_status.go

```diff
@@ -24,7 +33,25 @@ func (r *reconciler) syncIngressControllerStatus(deployment *appsv1.Deployment,
 	updated := ic.DeepCopy()
 	updated.Status.AvailableReplicas = deployment.Status.AvailableReplicas
 	updated.Status.Selector = selector.String()
-	updated.Status.Conditions = computeIngressStatusConditions(updated.Status.Conditions, deployment)
+
+	updated.Status.Conditions = []operatorv1.OperatorCondition{}
+	updated.Status.Conditions = append(updated.Status.Conditions, computeIngressStatusConditions(updated.Status.Conditions, deployment)...)
+	updated.Status.Conditions = append(updated.Status.Conditions, r.statusCache.computeLoadBalancerStatus(ic)...)
+
+	for i := range updated.Status.Conditions {
+		newCondition := updated.Status.Conditions[i]
+		var oldCondition *operatorv1.OperatorCondition
+		for j, possibleOldCondition := range ic.Status.Conditions {
+			if possibleOldCondition.Type == newCondition.Type {
+				old := ic.Status.Conditions[j]
+				oldCondition = &old
+				break
+			}
+		}
+		lastTransitionTime := getLastTransitionTime(&newCondition, oldCondition)
```

Comment:
setLastTransitionTime has an issue w/ modifying the array element pointer in this context, didn't take the time to look closely. Seemed easier to just remove the side effect

Reply:
Yes, that did it! thanks